test/ring/middleware/test/json.clj

```diff
@@ -337,6 +337,20 @@
       (is (= (get-in response [:headers "Content-Type"]) "application/json; some-param=some-value"))
       (is (= (:body response) "{\"foo\":\"bar\"}")))))
 
+(deftest test-json-response-streaming
+  (testing "streaming vector body"
+    (let [handler  (constantly {:status 200 :headers {} :body [:foo :bar]})
+          response ((wrap-json-response handler {:stream true}) {})]
+      (is (= (get-in response [:headers "Content-Type"]) "application/json; charset=utf-8"))
+      (is (= (slurp (:body response)) "[\"foo\",\"bar\"]")))) ;; using slurp checks that we have an inputstream
```

Comment:
No need for the comment here, as I think it's self-evident what `slurp` is being used for.